engine/src/main/java/org/terasology/rendering/nui/editor/NUIEditorScreen.java

```diff
@@ -124,8 +131,11 @@ public void initialise() {
                 .stream()
                 .collect(Collectors.toList()));
 
-        // Exclude the NUI editor itself, then sort the list.
-        availableAssetList.removeIf(asset -> asset.getRootUrn().equals(ASSET_URI));
+        // Exclude screens used by the NUI editor, then sort the list.
+        availableAssetList.removeIf(asset -> asset.getRootUrn().equals(this.ASSET_URI));
+        availableAssetList.removeIf(asset -> asset.getRootUrn().equals(ContextMenuScreen.ASSET_URI));
+        availableAssetList.removeIf(asset -> asset.getRootUrn().equals(NUIEditorSettingsScreen.ASSET_URI));
+        availableAssetList.removeIf(asset -> asset.getRootUrn().equals(WidgetSelectionScreen.ASSET_URI));
```

Comment:
The copies still get registered within `NUIManager` as the instance of a specific screen class, which means that `ContextMenuBuilder` instead of pushing a new context menu at a specific location [attempts to fetch](https://github.com/MovingBlocks/Terasology/blob/develop/engine/src/main/java/org/terasology/rendering/nui/contextMenu/ContextMenuBuilder.java#L72) the existing (deliberately uninitialized) `ContextMenuScreen` and causes a NPE. This isn't normally a problem unless meta-editing like this is attempted :smile: